src/clj/kamera/core.clj

```diff
@@ -147,7 +151,7 @@
                     (.getAbsolutePath actual)
                     (.getAbsolutePath difference)]
                    opts)
-           mean-absolute-error (when-let [e (last (re-find #"all: .* \((.*)\)" stderr))]
+           mean-absolute-error (when-let [e ((compare-parsers metric) stderr)]
```

Comment:
Might be a good idea to have a default too, so that we're not limiting the user to just the enumerated algos. Not sure which one would be better though, I guess the mae one would be best because people may already be using that one now with other algos.

Reply:
Also this was my oversight, but could we also rename the `mean-average-error` to be `metric`?

Reply:
Ok 👍